test/executor/test_load_executor.py

```diff
@@ -42,10 +43,11 @@ def test_should_call_opencv_reader_and_storage_engine_and_config(
         append_mock.assert_called_once_with(table_metainfo, data)
```

Comment:
We need to add a check that create_mock is called once with.
and even append_mock might change if we make changes in the load executor.

Reply:
Fixed by 79651a5